tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/CreatedumpTests.cs

```diff
@@ -288,6 +288,31 @@ public async Task NativeCrash()
         File.Exists(reportFile.Path).Should().BeTrue();
     }
 
+    [SkippableFact]
+    public async Task CheckThreadName()
+    {
+        // Test that threads prefixed with DD_ are marked as suspicious even if they have nothing of Datadog in the stacktrace
+
+        SkipOn.Platform(SkipOn.PlatformValue.MacOs);
```

Comment:
You don't need to skip on Windows too?

Reply:
Good question. Those tests don't run on windows, I suppose it's because they don't have the right attribute